source/fundamentals/indexes.txt

```diff
@@ -270,6 +270,11 @@ The following code example shows how to create an Atlas Search index:
 .. literalinclude:: /examples/generated/SearchIndexesTest.snippet.single-search-index-create.kt
    :language: kotlin
 
+To create multiple Search or Vector Search indexes, you must create a
```

Comment:
Q: Confirming that these should be capitalized

Reply:
I think so? seems like product names are capped https://www.mongodb.com/docs/meta/style-guide/style/product-names-version-numbers/
reference: https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/